aten/src/ATen/native/TensorIterator.h

```diff
@@ -134,6 +135,7 @@ enum class CommonDTypeStrategy : uint8_t {
 struct CAFFE2_API TensorIterator {
   using DimMask = std::bitset<64>;
   using PtrVector = SmallVector<char*, 4>;
+  using StrideVector = SmallVector<int64_t, 6>;
```

Comment:
super nit: maybe move this magical numbers into constant with good name
super super nit: make Ptr vector same size as StrideVector